src/YawPitchControl/input/RotationPanInput.js

```diff
@@ -64,10 +56,16 @@ export default class RotationPanInput extends PanInput {
 
 		const offset = super.getOffset(properties, [true, true]);
 		const newOffset = [0, 0];
-		const theta = this._screenRotationAngle.getRadian();
+
+		const rightAxis = this._deviceSensor.getDeviceHorizontalRight();
```

Comment:
@WoodNeck 

These are new codes, but getDeviceHorizontalRight is the only place to use it? Wouldn't it be better to use getRadian?

Reply:
For note: This method can be reused when we reenable pitch control on gyroMode="VR".